src/bsoncxx/document/element.hpp

```diff
@@ -69,6 +69,15 @@ class BSONCXX_API element {
 
     explicit operator bool() const;
 
+    const std::uint8_t * const get_raw() const;
```

Comment:
I would run clang format instead of fixing it by hand, but always glue the asterisk to the type name e.g.

`const std::uint8_t*`